src/app/collection/shared/included/curators/curators.component.scss

```diff
@@ -31,13 +31,14 @@
 }
 
 .curators {
-    color: $dark-grey;
+    color: transparent;
```

Comment:
Why was this adjusted?

Reply:
I was testing different sizing and wasn't seeing changes reflect so I changed the color here to see if that would transition, I then realized the color here doesn't really have an affect. I will double check though @pzalep1